src/components/navigation/nav-tab-panel.tsx

```diff
@@ -75,7 +76,7 @@ export class NavTabPanel extends BaseComponent<IProps, IState> {
                   })
                 }
               </TabList>
-              { user.isNetworkedTeacher
+              { (user.isNetworkedTeacher  && urlParams.chat)
```

Comment:
```suggestion
              { (user.isNetworkedTeacher && urlParams.chat)
```
More broadly, it seems like it might make sense to define `showChatColumn` at the top as something like:
```typescript
const showChatColumn = this.state.showChatColumn && (urlParams.chat !== undefined);
```
and then use the local `showChatColumn` down below. Note that the `!== undefined` formulation for the test allows `&chat` to enable the feature without having to type `&chat={something}` because parameters that are present without values get a value of `null` while parameters that aren't present get a value of `undefined`.